### PR TITLE
Translating forms.md

### PR DESCRIPTION
**Pull request by SahibArora**
Hi there, 
Sorry I started working on it, before your reply, just a little bit excited towards my first open source contribution. Hope you don't mind.
I am just opening a PR request for forms which I will be updating with time. When I will be done, I will let you know.

Thank you,

### CODE REVIEW

File: content/docs/forms.md [hunk 1]
```diff
@@ -9,23 +9,24 @@ redirect_from:
   - "docs/forms-zh-CN.html"
 ---
 
-HTML form elements work a little bit differently from other DOM elements in React, because form elements naturally keep some internal state. For example, this form in plain HTML accepts a single name:
+रियेक्ट मै HTML फॉर्म के तत्व DOM के तत्वो से थोड़ा भिन्न तरीके से काम करते है, क्योंकि फार्म के तत्व स्वाभाविक रूप से कुछ आंतरिक स्थिति रखते हैं, जैसे की, सादे HTML में यह फॉर्म एक ही नाम स्वीकार करेगा:
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: `मै` => `में`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `तत्व` => `एलिमेंट`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `तत्वो` => `एलिमेंट्स`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `फार्म` => `फॉर्म`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `आंतरिक स्थिति रखते हैं,` => `आंतरिक स्टेट रखते हैंैं ।`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `सादे HTML में यह फॉर्म एक ही नाम स्वीकार करेगा` => `यह फॉर्म सादे HTML में एक ही नाम स्वीकार करेगा`

File: content/docs/forms.md [hunk 2]
```diff
@@ -9,23 +9,24 @@ redirect_from:
...
 
 ```html
 <form>
   <label>
-    Name:
-    <input type="text" name="name" />
+    Name
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: `Name` => `Name:` 

do not change anything other than translating in the code example

File: content/docs/forms.md [hunk 3]
```diff
@@ -9,23 +9,24 @@ redirect_from:
...
+    <input  type ="text" name ="name" />
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: `<input  type ="text" name ="name" />`
remove unwanted extra spaces, keep it same as in english

File: content/docs/forms.md [hunk 4]
```diff
@@ -9,23 +9,24 @@ redirect_from:
...
   </label>
-  <input type="submit" value="Submit" />
+  <input  type ="submit" value  ="submit" />
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: same as above, keep it same as english

File: content/docs/forms.md [hunk 5]
```diff
@@ -9,23 +9,24 @@ redirect_from:
...
 </form>
 ```
 
-This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: remove this extra line break

File: content/docs/forms.md [hunk 6]
```diff
@@ -9,23 +9,24 @@ redirect_from:
...
 
-## Controlled Components {#controlled-components}
+जब उपयोगकर्ता फॉर्म को सबमिट करता है, तो इस फ़ॉर्म में नए पेज पर ब्राउज़िंग का डिफ़ॉल्ट HTML फॉर्म व्यवहार होता है। यदि आप इस व्यवहार को प्रतिक्रिया में चाहते हैं, तो यह काम करता है। लेकिन ज्यादातर मामलों में, जावास्क्रिप्ट फ़ंक्शन का उपयोग करना सुविधाजनक होता है जो फ़ॉर्म को प्रस्तुत करने में सक्षम होता है और उस डेटा तक पहुंच होती है जो उपयोगकर्ता ने फॉर्म में दर्ज किया था। इसे प्राप्त करने का मानक तरीका "नियंत्रित घटकों" नामक एक तकनीक के साथ है।
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: `फ़ॉर्म` => `फॉर्म` 

try to keep same spelling of same words

Comment (arshadkazmi42) [inline content/docs/forms.md]: `तो इस फ़ॉर्म में नए पेज पर ब्राउज़िंग का डिफ़ॉल्ट HTML फॉर्म व्यवहार होता है।`

does not makes any sense, can rephrase it to a meaningful sentence

Comment (arshadkazmi42) [inline content/docs/forms.md]: `प्रतिक्रिया` => `React`

we are not translating react, you can check the list of words which we are not translating at #2

Comment (arshadkazmi42) [inline content/docs/forms.md]: `घटकों` => `कौम्पोनॅन्ट`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `जावास्क्रिप्ट फ़ंक्शन का उपयोग करना सुविधाजनक होता है जो फ़ॉर्म को प्रस्तुत करने में सक्षम होता है और उस डेटा तक पहुंच होती है जो उपयोगकर्ता ने फॉर्म में दर्ज किया था। इसे प्राप्त करने का मानक तरीका "नियंत्रित घटकों" नामक एक तकनीक के साथ है।` 

whole sentence sounds a bit wrong, can check it once.

File: content/docs/forms.md [hunk 7]
```diff
@@ -9,23 +9,24 @@ redirect_from:
...
 
-In HTML, form elements such as `<input>`, `<textarea>`, and `<select>` typically maintain their own state and update it based on user input. In React, mutable state is typically kept in the state property of components, and only updated with [`setState()`](/docs/react-component.html#setstate).
+## नियंत्रित  अवयव {#controlled-components}
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: `नियंत्रित  अवयव` => `कंट्रोल्ड कौम्पोनॅन्ट`

File: content/docs/forms.md [hunk 8]
```diff
@@ -9,23 +9,24 @@ redirect_from:
...
+
+HTML में, फॉर्म के तत्व जैसे कि `<input>`, `<textarea>`, और `<select>` आमतौर पर अपनी स्वयं की स्थिति बनाए रखते है और उपयोगकर्ता इनपुट के आधार पर इसे अपडेट करते है. रियेक्ट के अंदर, स्टेट को ज्यादातर अवयव की स्टेट प्रोपोपेर्टी के अंदर ही रखा जाता है, और उसे सिर्फ [`setState()`](/docs/react-component.html#setstate) के दवारा ही अपडेट किया जा सकता है|
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: `तत्व` => `एलिमेंट`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `उपयोगकर्ता इनपुट` => `उपयोगकर्ता के इनपुट`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `रियेक्ट` => `React`

not translating react

Comment (arshadkazmi42) [inline content/docs/forms.md]: `ज्यादातर अवयव` => `ज्यादातर कौम्पोनॅन्ट`

Comment (arshadkazmi42) [inline content/docs/forms.md]: `प्रोपोपेर्टी` => `प्रॉपर्टी`

File: content/docs/forms.md (hunk 6 above)
Comment (SahibArora) [reply content/docs/forms.md]: please double check, as it is process
Thanks,

File: content/docs/forms.md (hunk 1 above)
Comment (arshadkazmi42) [reply content/docs/forms.md]: `रियेक्ट` => `React`

File: content/docs/forms.md (hunk 6 above)
Comment (arshadkazmi42) [reply content/docs/forms.md]: > please double check, as it is process
> Thanks,

I didn't get what do you mean by this? or what you want me to double check?

Comment (arshadkazmi42) [reply content/docs/forms.md]: > please double check, as it is process
> Thanks,

I didn't get what do you mean by this? or what you want me to double check?

**Comment by SahibArora** [comment]
Hi there,
Thanks for the suggestions. 
I did the changes, you asked me.
Please give it a look and then I will continue further.
Thanks,

File: content/docs/forms.md (hunk 4 above)
Comment (arshadkazmi42) [reply content/docs/forms.md]: You missed this one.
`Submit` should not be changed.
As value, `S` is capital

File: content/docs/forms.md [hunk 9]
```diff
@@ -9,23 +9,23 @@ redirect_from:
...
     Name:
     <input type="text" name="name" />
   </label>
-  <input type="submit" value="Submit" />
+  <input type="submit" value="submit" />
 </form>
 ```
 
-This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
+ऊपर दिया गया फॉर्म अन्य HMTL फॉर्म की तरह सबमिट करने पर नए पेज पर चला जाता है। यदि आप इस व्यवहार को प्रतिक्रिया में चाहते हैं, तो यह काम करता है। लेकिन ज्यादातर मामलों में, जावास्क्रिप्ट फ़ंक्शन का उपयोग करना सुविधाजनक होता है क्युकी वह फॉर्म को सब्मिट करने के साथ साथ उसकी उस डाटा तक भी पहुँच होती है जो यूजर फॉर्म मैं भरता है। इसे प्राप्त करने का मानक तरीका "कंट्रोल्ड कौम्पोनॅन्ट" नामक एक तकनीक के साथ है।
```
Comment (arshadkazmi42) [inline content/docs/forms.md]: `प्रतिक्रिया` => `React`

File: content/docs/forms.md (hunk 7 above)
Comment (arshadkazmi42) [reply content/docs/forms.md]: This is not fixed yet

**Comment by SahibArora** [comment]
I did the changes, thanks for your suggestions again.
I will carry on with the translating.
Thanks!

File: content/docs/forms.md (hunk 4 above)
Comment (arshadkazmi42) [reply content/docs/forms.md]: @SahibArora any updates on this?

Comment (SahibArora) [reply content/docs/forms.md]: Hi there, 
I was a bit busy from two weeks, I will continue to work on it before end of this week. Thanks,

**Comment by arshadkazmi42** [comment]
@SahibArora There is one line left for translation line 30, once you can do the translation of that line, I can review and approve it for second round of review.

**Comment by arshadkazmi42** [comment]
@SahibArora Any updates on this? Let us know if you need any help from our end

**Comment by arshadkazmi42** [comment]
@SahibArora Any updates on this? Let us know if you need any help from our end

**Comment by arshadkazmi42** [comment]
@SahibArora Any updates?

**Comment by saranshkataria** [comment]
@SahibArora please let us know if you will be working on this?

**Comment by SahibArora** [comment]
Hi @arshadkazmi42, 
I apologize for the delay and not been able to follow-up to your past messages. 
I will go through it right away and submit the file as soon as possible.
Sorry again for the inconvenience caused because of me.

**Comment by saranshkataria** [comment]
@SahibArora no worries. Just wanted to ensure that you will complete it. Let us know if you need any help

**Comment by saranshkataria** [comment]
@SahibArora any updates?

**Comment by saranshkataria** [comment]
Closing this PR due to inactivity. Feel free to reopen if you are going to work on it in the future.
